### PR TITLE
Add missing react/-native imports to example code

### PR DESCRIPTION
**Pull request by craigayre**

### CODE REVIEW

File: components/basics/react-native.js [hunk 1]
```diff
@@ -6,6 +6,8 @@ import Note from '../Note';
 import Link from '../Link';
 
 const sample = (`
+import React from 'react';
+import { Text, View } from 'react-native';
```
Comment (philpl) [inline components/basics/react-native.js]: This `react-native` import is actually not necessary :wink:

**Comment by SaraVieira** [comment]
@philpl @morajabi Is this PR still valid ?

**Comment by morajabi** [comment]
@SaraVieira I don't think so, and there's no such path anymore, but we need to add that React import here https://github.com/styled-components/styled-components-website/blob/master/sections/basics/react-native.js#L10
```js
import React from 'react'
```

**Comment by SaraVieira** [comment]
closed by https://github.com/styled-components/styled-components-website/pull/125

**Comment by philpl** [comment]
Alright I’ll close this for now. Once I add custom transpilers we could use Babel-standalone and traverse the AST to remove import statements, and add them to all examples

**Pull request closed without merging**